packages/twenty-front/src/utils/displaySubFieldsAddress.ts

```diff
@@ -0,0 +1,30 @@
+import { FieldAddressValue } from '@/object-record/record-field/types/FieldMetadata';
+import { isNonEmptyString } from '@sniptt/guards';
+import {
+  ALLOWED_ADDRESS_SUBFIELDS,
+  AllowedAddressSubField,
+} from 'twenty-shared/src/types/AddressFieldsType';
+import { isDefined } from 'twenty-shared/utils';
+
+export const extractSubFieldsAddressValues = (
```

Comment:
1 util per file + could be named formatAddressDisplay / joinAddressFieldValues for example